### ISSUE TITLE
Can't use PlumberStatic

### ISSUE THREAD

**Issue by chenjr0719** [trestletech/plumber#156]
As the [documents](https://www.rplumber.io/docs/programmatic-usage.html#mount-static) says, plumber can use PlumberStatic to add and mount new static file router now.

But I can't use PlumberStatic, would you please do me a favor to solve this problem?

BTW, my plumber version is 0.4.2.

**Comment by trestletech** [comment]
Hi. Can I get more details on what's going wrong?

The docs you referenced are specifically for building an API programatically. If you're using the more conventional format of commenting on your functions, you might be looking for https://www.rplumber.io/docs/routing-and-input.html#static-file-handler instead

e.g. adding something like 

```
#* @assets ./files/static /static
list()
```

should make the `./files/static` folder accessible at the `/static` URL on your API.

**Comment by chenjr0719** [comment]
The directory structure is:
![image](https://user-images.githubusercontent.com/13619684/29597001-76fcaab0-87f2-11e7-9532-04f9a2bca1bd.png)

I try to run the code in doc and get the error like this:

![image](https://user-images.githubusercontent.com/13619684/29597024-9fd383dc-87f2-11e7-889e-d9cb2032d149.png)

That's because `PlumberStatic` doesn't available in this package
![image](https://user-images.githubusercontent.com/13619684/29596439-bf2c5c16-87ef-11e7-99fe-267d21d69ccd.png)

Before I reply this issue, I clone this repository and go to check the [NAMESPACE](https://github.com/trestletech/plumber/blob/master/NAMESPACE),  it seems that the function [`PlumberStatic`](https://github.com/trestletech/plumber/blob/master/R/plumber-static.R) not to be exported.

After add the comment `#' @export` at `PlumberStatic` and reinstall package, it appeared:
![image](https://user-images.githubusercontent.com/13619684/29596383-761f8b6a-87ef-11e7-9629-7490e17f727f.png)

It should be worked, but actually not, I get 404 error when I visit http://127.0.0.1:8080/assets:
![image](https://user-images.githubusercontent.com/13619684/29597041-c33a8488-87f2-11e7-9f79-8e10abf529f2.png)

BTW, I also try your suggestion, but no luck:
![image](https://user-images.githubusercontent.com/13619684/29597152-5c2959d0-87f3-11e7-8b3a-817f7640d403.png)

Here is my **plumber.R**:
```
#* @assets ./myfiles /assets
list()
```

**Comment by trestletech** [comment]
Doh! Sorry, the export should be solved in #163.

I think the problem you're seeing now is related to the fact that Plumber doesn't offer directory indexes (would be a nice feature). So you won't see anything at the root of the directory, but you should see something if you go to: `localhost:8080/assets/test.txt`

**Comment by chenjr0719** [comment]
Oops, you are right. I can see something when I visit http://localhost:8080/assets/test.txt. That's my mistake and thanks for you to help me figure it out.

And the problem of `PlumberStatic` is also solved, this issue should be close.
